modules/installation-special-config-storage.adoc

```diff
@@ -18,25 +35,46 @@ This feature:
 * Requires no user intervention for providing passphrases
 * Uses AES-256-XTS encryption, or AES-256-CBC if FIPS mode is enabled
 
-There are two different supported encryption modes:
+[id="installation-special-config-encryption-threshold_{context}"]
+=== Configuring an encryption threshold
+
+In {product-title}, you can specify a requirement for more than one Tang server. You can also configure the TPM v2 and Tang encryption modes simultaneously, so that the boot disk data can be decrypted only if the TPM secure cryptoprocessor is present and the Tang servers can be accessed over a secure network. 
 
-* TPM v2: This is the preferred mode. TPM v2 stores passphrases in a secure cryptoprocessor.
-* Tang: To use Tang to encrypt your cluster, you need to use a Tang server. Clevis implements decryption on the client side.
+You can use the `threshold:` attribute in your Butane configuration to define the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur. The threshold is met when the stated value is reached through any combination of the declared conditions. For example, the `threshold:` value of `2` in the following configuration can be reached by accessing the two Tang servers, or by accessing the TPM secure cryptoprocessor and one of the Tang servers:
+
+.Example Butane configuration
```

Comment:
This isn't a full config and will fail validation.  That's fine, but maybe we should use a term like `configuration fragment` to make it clearer.

Reply:
@bgilbert thank you for reviewing the changes and for highlighting this. I have now updated the example so that it is a valid disk encryption config. I have also updated the title to "Example Butane configuration for disk encryption", so that it is a little more descriptive.